dfloat11_custom.py

```diff
@@ -431,6 +431,14 @@ def __init__(self, model, load_device, offload_device, size=0, weight_inplace_up
         super().__init__(model, load_device, offload_device, size=size, weight_inplace_update=weight_inplace_update)
         self.model.state_dict = patch_state_dict(self.model.state_dict)
     
+    def partially_unload(self, offload_device, memory_to_free=0):
+        """
+        Override to disable partial unloading for DFloat11 models.
+        DFloat11 models use a custom compressed weight format that is 
+        incompatible with ComfyUI's standard partial unloading mechanism.
+        The DFloat11 system already handles its own memory management through CPU offloading.
+        """
+        return 0
```

Comment:
I guess this is one of the things that broke due to the most recent ComfyUI updates, which I was reluctant to try out because I heard the manager just completely stopped working. 

But I guess a simple `return 0` should be fine for now, until I get to test it out more thoroughly and see if I can improve upon it.

Reply:
I think your docstring comment is not really correct, because the CPU offloading feature you mentioned is for block-swapping when the full DFloat11 model cannot fit into VRAM, but I believe (correct me if I am wrong here, since this pertains to ComfyUI core) the `partially_unload` feature is for unloading part of the diffusion model after it is done, to free up VRAM for loading the text encoder (in preparation for processing the next prompt)